code/script.js

```diff
@@ -1,325 +1,1505 @@
 // All the DOM selectors stored as short variables
-const board = document.getElementById('board')
-const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+const board = document.getElementById(`board`);
+const questions = document.getElementById(`questions`);
+const restartBtn = document.getElementById(`restart`);
+const findOutBtn = document.getElementById(`filter`);
+const winOrLose = document.getElementById(`winOrLose`);
+const championCounter = document.getElementById(`championCounter`);
+const playAgain = document.getElementById(`playAgain`);
+const questionsHistory = document.getElementById(`questionsHistory`);
 
 // Array with all the characters, as objects
 const CHARACTERS = [
   {
-    name: 'Jabala',
```

Comment:
omg so impressed by your characterlist! How you had the energy to make this huge list to choose characters from and to keep all the pictures in order!